src/views/navigation/communityList.js

```diff
@@ -40,8 +41,15 @@ const CommunityListItem = props => {
 
   const appControlSymbol = '⌘';
 
+  const isWideViewport =
+    window && window.innerWidth > MIN_WIDTH_TO_EXPAND_NAVIGATION;
```

Comment:
This will break SSR rehydration any logged-in page I think? We should do this with CSS media queries inside the `Tooltip` component.